src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs

```diff
@@ -52,5 +63,7 @@ internal class ProjectRestorer : IProjectRestorer
                                                  result.StdErr + result.StdOut);
             }
         }
+
+        private ICommandFactory _commandFactory;
```

Comment:
NIT: we usually put these at the top.

Reply:
I'll fix.  I was secretly hoping my continued sneaking this in would inevitably become the defacto standard and we'd have to change the .NET guidelines :)

Reply:
Hank eyes.